ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -1216,113 +1226,113 @@ def test027_update_material_type(self):
     # will continue with us & then put the test case number for it 
     def test026_update_suborder_article(self):
         self.base_selenium.LOGGER.info('Order created with 4 suborders with the following data')
-        self.base_selenium.LOGGER.info('Material type: {}, Article name: {}, Test plans: {}, Test Units: {}'.format(
-            suborder_data['material_types'], suborder_data['article'], suborder_data['test_plan'],
-            suborder_data['test_unit']))
+        #self.base_selenium.LOGGER.info('Material type: {}, Article name: {}, Test plans: {}, Test Units: {}'.format(
```

Comment:
why you Comment this test case in this PR?

Reply:
because a lot of old order test cases have syntax error and you will find a lot of them already comment because we now re-implement them but anyways I removed the comment ( but both with & without comment no different )